.github/workflows/require-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Require Tests'
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
For consistency with the other workflows, please add a blank line above this line, and also please provide a `name` for this job.